helix-term/src/handlers/completion/resolve.rs

```diff
@@ -42,10 +42,30 @@ impl ResolveHandler {
         if item.resolved {
             return;
         }
-        let needs_resolve = item.item.documentation.is_none()
-            || item.item.detail.is_none()
-            || item.item.additional_text_edits.is_none();
-        if !needs_resolve {
+        // We consider an item to be fully resolved if it has non-empty, none-`None` details,
```

Comment:
is `none-` supposed to be `non-`? Or is this implying something else?

Reply:
Yep that's a typo, should be "non-`None`"